spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -195,41 +198,43 @@ public static AmqpAppender createAppender(// NOSONAR NCSS line count
 			theLayout = PatternLayout.createDefaultLayout();
 		}
 		AmqpManager manager = new AmqpManager(configuration.getLoggerContext(), name);
-		manager.uri = uri;
-		manager.host = host;
-		Optional.ofNullable(port).ifPresent(v -> manager.port = Integers.parseInt(v));
-		manager.addresses = addresses;
-		manager.username = user;
-		manager.password = password;
-		manager.virtualHost = virtualHost;
-		manager.useSsl = useSsl;
-		manager.verifyHostname = verifyHostname;
-		manager.sslAlgorithm = sslAlgorithm;
-		manager.sslPropertiesLocation = sslPropertiesLocation;
-		manager.keyStore = keyStore;
-		manager.keyStorePassphrase = keyStorePassphrase;
-		manager.keyStoreType = keyStoreType;
-		manager.trustStore = trustStore;
-		manager.trustStorePassphrase = trustStorePassphrase;
-		manager.trustStoreType = trustStoreType;
-		manager.senderPoolSize = senderPoolSize;
-		manager.maxSenderRetries = maxSenderRetries;
-		manager.applicationId = applicationId;
-		manager.routingKeyPattern = routingKeyPattern;
-		manager.generateId = generateId;
-		manager.deliveryMode = MessageDeliveryMode.valueOf(deliveryMode);
-		manager.exchangeName = exchange;
-		manager.exchangeType = exchangeType;
-		manager.declareExchange = declareExchange;
-		manager.durable = durable;
-		manager.autoDelete = autoDelete;
-		manager.contentType = contentType;
-		manager.contentEncoding = contentEncoding;
-		manager.connectionName = connectionName;
-		manager.clientConnectionProperties = clientConnectionProperties;
-		manager.charset = charset;
-		manager.async = async;
-		manager.addMdcAsHeaders = addMdcAsHeaders;
+		JavaUtils.INSTANCE
```

Comment:
`JavaUtils` is there since `2.1.x`. So, this is not going to be clearly back-ported into `2.0.x` and `1.7.x`.
However I can play with `1.7.x` if you confirm that we don't back-port to `2.0.x` because of its EOL 😄

Reply:
Right; just 2.1.x - it turns out SSL support wasn't added until 2.0 anyway https://jira.spring.io/browse/AMQP-624

I've repushed with a modified commit comment.